rocketmq-connect-hudi/src/main/java/example/avro/User.java

```diff
@@ -0,0 +1,501 @@
+package example.avro; /**
+ * Autogenerated by Avro
+ *
+ * DO NOT EDIT DIRECTLY
+ */
+
+import org.apache.avro.message.BinaryMessageDecoder;
+import org.apache.avro.message.BinaryMessageEncoder;
+import org.apache.avro.message.SchemaStore;
+import org.apache.avro.specific.SpecificData;
+import org.apache.avro.util.Utf8;
+
+@org.apache.avro.specific.AvroGenerated
+public class User extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
```

Comment:
Is this file necessary? If it is a necessary class, please modify the package name and include the apache license

Reply:
removed